organization/components/UserTable.tsx

```
@@ -0,0 +1,88 @@
+            </tr>
+          </thead>
+          <tbody className="divide-y divide-gray-200">
+            {users.map(user => (
+              <tr key={user._id} className="hover:bg-gray-50 transition-colors">
+                <td className="whitespace-nowrap py-4 pl-6 pr-3 text-sm text-gray-900">
+                  {user.first_name + ' ' + user.last_name}
+                </td>
+                <td className="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
+                  {user.email}
+                </td>
+                <td className="whitespace-nowrap px-3 py-4 text-sm text-gray-500"></td>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)